Altis_Life.Altis/stringtable.xml

```diff
@@ -1564,7 +1564,7 @@
             <Czech>Máte neplatné znaky v názvu vaší gangu. To se může skládat pouze z číslic a písmen s podtržítkem</Czech>
             <Spanish>El nombre de tu pandilla solo puede consistir de: "Numeros, Letras y guion bajo"</Spanish>
             <Russian></Russian>
-            <German>Du hast ein ungültiges Zeichen in deinem Gang Namen. Der Name darf nur aus Zahlen und Buchstaben sowie einem Unterstrich bestehen</German>
+            <German>Du hast ein ungültiges Zeichen in deinem Gang-Namen. Der Name darf nur aus Zahlen und Buchstaben sowie einem Unterstrich bestehen.</German>
```

Comment:
`Du hast ein ungültiges Zeichen in deinem Gangnamen. Der Name darf nur aus Zahlen und Buchstaben sowie einem Unterstrich bestehen.`